### ISSUE TITLE
python3 (guess)

### ISSUE THREAD

**Issue by luchmhor** [fayer3/plugin.video.zappntv#4]
hey, i get following error when using the app:

```
2020-03-13 16:00:16.489 T:2141   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'TypeError'>
                                            Error Contents: the JSON object must be str, not 'bytes'
                                            Traceback (most recent call last):
                                              File "/home/osmc/.kodi/addons/plugin.video.zappntv/main.py", line 14, in <module>
                                                plugin.run()
                                              File "/home/osmc/.kodi/addons/plugin.video.zappntv/resources/lib/plugin.py", line 1153, in run
                                                plugin.run()
                                              File "/home/osmc/.kodi/addons/script.module.routing/lib/routing.py", line 130, in run
                                                self._dispatch(self.path)
                                              File "/home/osmc/.kodi/addons/script.module.routing/lib/routing.py", line 141, in _dispatch
                                                view_func(**kwargs)
                                              File "/home/osmc/.kodi/addons/plugin.video.zappntv/resources/lib/plugin.py", line 87, in index
                                                icon_json = json.loads(icon)
                                              File "/usr/lib/python3.5/json/__init__.py", line 312, in loads
                                                s.__class__.__name__))
                                            TypeError: the JSON object must be str, not 'bytes'
                                            -->End of Python script error report<--
2020-03-13 16:00:16.794 T:2140   ERROR: GetDirectory - Error getting plugin://plugin.video.zappntv/
2020-03-13 16:00:16.796 T:780   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.zappntv/) failed
```

I'm using osmc development branch

**Comment by fayer3** [comment]
the newest kodi 19 alpha versions for windows use python 3.7 where bytes is allowed.

but seems like python 3.5 doesn't allow that. Will be fixed in the next version.

**Comment by luchmhor** [comment]
perfekt, thanks.

**Comment by fayer3** [comment]
should be fixed in version 1.3.2.
